packages/components/src/components/SquigglePlayground/index.tsx

```diff
@@ -170,13 +174,15 @@ export const SquigglePlayground: React.FC<PlaygroundProps> = (props) => {
     squiggleChart
   );
 
+  const standardHeightStyle = { height: `${height}px`, overflow: "auto" };
```

Comment:
(minor)

In React, numbers as width and height work fine, so this could be `{ height, ... }`.

Also, it'd be good to type this as `CSSProperties`.